gluster-exporter/metric_ps.go

```diff
@@ -199,36 +197,36 @@ func ps() {
 		}
 
 		if err != nil {
```

Comment:
this condition can be removed since by default unknown labels are assigned

Reply:
Ack